### ISSUE TITLE
[references] Add slack logging & fix clearml integration

### ISSUE THREAD

**Issue by felixdittrich92** [mindee/doctr#1458]
### 🚀 The feature

- Add option to use slack for logging with `tqdm` in each training script
- The logs should be clear distinguishable if you log multible runs at onces to the same channel

- [x] Slack
- [x] ClearML 

- (Optional): It would be great to track  / log the learning rate also after each epoch :)

**Comment by odulcy-mindee** [comment]
I'd be a little disappointed to remove the ClearML integration because I think it's a very good tool, but unfortunately there's a memory leak that's very hard to find. For the moment, we can remove this integration but I'd like to see it added again, I think it's a very good feature.

**Comment by odulcy-mindee** [comment]
For the slack logging, I already built something internally. As discussed, there are also other tools available, it should easy to integrate that.

**Comment by felixdittrich92** [comment]
Sounds good to me then let us split this ticket into two parts one for slack and the other for ClearML 👍

**Comment by odulcy-mindee** [comment]
Slack has been done in https://github.com/mindee/doctr/pull/1837

**Comment by felixdittrich92** [comment]
#1844
